### ISSUE TITLE
Plugin conflict: I had to deactivate new Elementor 3.1 because it interfere with TUTOR LMS plugin.

### ISSUE THREAD

**Issue by FabQuimbaya** [elementor/elementor#13706]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by FabQuimbaya** [comment]
The entire training course on our website, made with the TUTOR LMS plugging disappeared after Elementor 3.1 update.
I deactivated Elementor and all the courses are visible again.

**Comment by shilo-ey** [comment]
Thanks!

We are looking into this,


For now, in order to overcome this issue - please go to Elementor > Settings > Experiments and turn Landing Pages experiment off.

We will soon resolve this conflict and update here.

Thanks

**Comment by FabQuimbaya** [comment]
I've just done as you said, and it works.
Thank you!

**Comment by deborahfelicio** [comment]
hi guys, I am having the same issues with LearnDash, turning the Landing Pages experiment off fixed the issues for now.

**Comment by DorShahar** [comment]
Thanks @deborahfelicio  - we will check the conflicts with this plugin as well.

**Comment by udidol** [comment]
Hi @deborahfelicio,
We would like to investigate your issue.
In order for us to be able to do that, could you open a new issue describing your problem with LearnDash? 
We would appreciate if you could share your system information and include steps to reproduce the issue.

**Comment by deborahfelicio** [comment]
done now here https://github.com/elementor/elementor/issues/13725

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.1.1

Feel free to update

Thanks!

**Comment by markwmim** [comment]
I have confirmed it is working properly in 3.1.1. 

Thanks!
